### ISSUE TITLE
System beep played when MenuBarItem AccessKeys are invoked while holding down the ALT key

### ISSUE THREAD

**Issue by rfrancioso-gpsw** [microsoft/microsoft-ui-xaml#8554]
### Describe the bug

For the top-level MenuBar of my application, when I press an AccessKey while **holding** down the `ALT` key, the system plays a warning beep sound, telling me that something has gone wrong.

### Steps to reproduce the bug

The following basic markup can be used to reproduce the bug.  Simply paste it into `MainWindow.xaml` after creating a new C# WinAppSDK application:

```xml
<Grid>
    <MenuBar VerticalAlignment="Top">
        <MenuBarItem Title="File" AccessKey="F">
            <MenuFlyoutItem Text="Open" Icon="OpenFile" />
        </MenuBarItem>
    </MenuBar>
</Grid>
```

### Expected behavior

When I press `ALT+F` (both keys **together**), this is supposed to invoke the MenuBarItem that has `F` assigned as an `AccessKey`.  It does do this (the menu item opens, and I see the subitem flyout pop open underneath it).  However, for some reason, it also results in the system playing an error sound (the same one that's played when trying to click outside of a modal window).  That shouldn't be happening.

### NuGet package version

None

### Additional context

Using `Microsoft.WindowsAppSDK v1.2.230217.4`.

**Comment by wbokkers** [comment]
Yes, I find this very annoying as well. There is no way to suppress the beep.

**Comment by yaira2** [comment]
We're experiencing this issue in Files https://github.com/files-community/Files/issues/14743
